### ISSUE TITLE
Some windows users are unable to use Lantern because "An attempt was made to access a socket in a way forbidden by its access permissions"

### ISSUE THREAD

**Issue by oxtoacart** [getlantern/lantern#4296]
Per [this article](http://answers.microsoft.com/en-us/ie/forum/ie10-windows_8/an-attempt-was-made-to-access-a-socket-in-a-way/4a955300-4cc6-4b2f-b426-6d6f07e4eddc?auth=1) it could be caused by a firewall or antivirus software.

**Comment by percyalpha** [comment]
The user said he used F-Secure Client Security. And after whitelisting Lantern.exe, he is able to connect normally.

**Comment by percyalpha** [comment]
Another user said the problem is resolved after whitelisting in Norton anti-virus.

**Comment by oxtoacart** [comment]
Thanks @percyalpha!

cc: @sunshine19090 who has been coordinating our whitelisting with antivirus vendors.

**Comment by sunshine19090** [comment]
Oh, we need to submit new version of our files to norton as they whitelist the specific version that is sent. I will send the 2.3.0 files now. Thanks @oxtoacart and @percyalpha. Yeah, the other way around is that the user can manually whitelist the file in their system but it may turn unsafe if they have downloaded from other sites and not directly from ours.

**Comment by myleshorton** [comment]
2.3.0 isn't out yet @sunshine19090, so I would send the 2.2.3 files.

On Mon, May 16, 2016 at 11:22 AM, sunshine19090 <notifications@github.com>
wrote:

> Oh, we need to submit new version of our files to norton as they whitelist
> the specific version that is sent. I will send the 2.3.0 files now. Thanks
> @oxtoacart <https://github.com/oxtoacart> and @percyalpha
> <https://github.com/percyalpha>. Yeah, the other way around is that the
> user can manually whitelist the file in their system but it may turn unsafe
> if they have downloaded from other sites and not directly from ours.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/getlantern/lantern/issues/4296#issuecomment-219503734>
>



-- 
--
President
Brave New Software Project, Inc.
https://www.getlantern.org
pgp A998 2B6E EF1C 373E 723F A813 045D A255 901A FD89

**Comment by sunshine19090** [comment]
@oxtoacart @percyalpha Also, Can I have a list of Anti-virus softwares that the users use widely so that I can work with them first and then move onto others?

**Comment by sunshine19090** [comment]
Got it @myleshorton ! I will send the 2.2.3 files.

**Comment by percyalpha** [comment]
@sunshine19090 Most users use 360 anti-virus but I assume you meant foreign anti-virus. So Kaspersky, Avast, Avria, Norton, Mcafee, Microsoft, AVG

**Comment by sunshine19090** [comment]
Thanks @percyalpha. Can you provide me the link for 360 anti-virus? I see only 360Total security in my search results. Is that what it is?

**Comment by percyalpha** [comment]
Yeah, but don't bother with that. Because that's the foreign version which is completely different from the domestic version

**Comment by percyalpha** [comment]
And assume all the information you submitted to them is shared with the gov

**Comment by sunshine19090** [comment]
No @percyalpha, I haven't shared anything. I just submit forms and follow up with the AV companies. Why do we have to share it with the gov anyway? Will that have any impact?

**Comment by percyalpha** [comment]
I just tested with 360 anti-virus. So far, they're not flagged us. So let's not contact them. We don't need to share anything with the gov. I was just saying that if you share anything with them, you might as well share it with the gov.

**Comment by sunshine19090** [comment]
This should be good now in Norton. I guess the user may have to update Norton before trying again with Lantern 2.2.3.
